src/abstractdataframe/show.jl

```diff
@@ -556,32 +574,77 @@ end
 #' df = DataFrame(A = 1:3, B = ["x", "y", "z"])
 #' showall(df, true)
 function Base.showall(df::AbstractDataFrame,
-                      splitchunks::Bool = false) # -> Void
-    showall(STDOUT, df, splitchunks)
+                      splitchunks::Bool = false,
+                      onechunk::Bool = false) # -> Void
+    showall(STDOUT, df, splitchunks, onechunk)
     return
 end
 
+#' @exported
 #' @description
 #'
 #' Render a summary of the column names, column types and column missingness
 #' count.
 #'
 #' @param io::IO The `io` to be rendered to.
 #' @param df::AbstractDataFrame An AbstractDataFrame.
+#' @param allcols::Bool If `false` (default) only a subset of columns
+#'        fitting on the screen is printed
+#' @param values::Bool If `true` (default) first and last value of
+#'        each column is printed
 #'
 #' @returns o::Void A `nothing` value.
 #'
 #' @examples
 #'
 #' df = DataFrame(A = 1:3, B = ["x", "y", "z"])
-#' showcols(df, true)
-function showcols(io::IO, df::AbstractDataFrame) # -> Void
+#' showcols(STDOUT, df)
+function showcols(io::IO, df::AbstractDataFrame, allcols::Bool = false, values::Bool = true) # -> Void
     println(io, summary(df))
     metadata = DataFrame(Name = _names(df),
                          Eltype = eltypes(df),
                          Missing = colmissing(df))
-    showall(io, metadata, true, @compat(Symbol("Col #")), false)
+    nrows, ncols = size(df)
+    if values && nrows > 0
+        if nrows == 1
+            metadata[:Values] = [Symbol(df[1, i]) for i in 1:ncols]
```

Comment:
`sprint` would make `:Values` column a string and thus it would be rendered with additional enclosing double quotes when printed and symbol is printed "as is". Your suggestion is valuable though :) and I have modified the code.